crates/swc_ecma_minifier/src/compress/optimize/mod.rs

```diff
@@ -2578,6 +2578,37 @@ impl VisitMut for Optimizer<'_> {
         debug_assert_eq!(self.prepend_stmts.len(), prepend_len);
         debug_assert_eq!(self.append_stmts.len(), append_len);
         debug_assert_valid(s);
+
+        if let Stmt::Block(block) = s {
```

Comment:
Does it need to be in the full optimizer?
It seems like it's not using any field from `self`

Reply:
No. It's just because the original code is in optimize